packages/api-reference/src/features/toolbar/ApiReferenceToolbar.vue

```diff
@@ -0,0 +1,38 @@
+<script lang="ts" setup>
+import type { ApiReferenceConfiguration } from '@scalar/types'
+import { computed } from 'vue'
+
+import ApiReferenceToolbarConfig from '@/features/toolbar/ApiReferenceToolbarConfig.vue'
+import ApiReferenceToolbarSdks from '@/features/toolbar/ApiReferenceToolbarSdks.vue'
+import ApiReferenceToolbarShare from '@/features/toolbar/ApiReferenceToolbarShare.vue'
+
+/** Turn this on to enabled the localhost toolbar */
+const FEATURE_FLAG = false as const
+
+defineProps<{
+  configuration?: Partial<ApiReferenceConfiguration>
+}>()
+
+const overrides = defineModel<Partial<ApiReferenceConfiguration>>('overrides')
+
+const showToolbar = computed<boolean>(() => {
+  if (!window || !FEATURE_FLAG) {
+    return false
+  }
+  return window.location.hostname === 'localhost'
```

Comment:
there is a wonderful helper that we can use here

https://github.com/scalar/scalar/blob/023f9d1ef70a66f27286a758a1ac6677a9d1ddc2/packages/helpers/src/url/is-local-url.ts

Reply:
Done!